tool/src/java/org/sakaiproject/attendance/tool/pages/EventView.java

```diff
@@ -293,6 +295,9 @@ public Component getLazyLoadComponent(String s){
                 };
                 item.add(dataPanelSlow);
             }
-        });
+        };
+        recordDataView.setItemsPerPage(25);
```

Comment:
Can we do serverConfigurationService.getInt("attendance.pager.items", 25)

Reply:
Great suggestion. I've made it configurable but default 25